pubspec.yaml

```diff
@@ -1,27 +1,27 @@
 name: flutter_key_value_file_storage
 description: An implementation to save big chunks of content to a file
-version: 0.0.7
+version: 0.0.9
```

Comment:
maybe change the version to 0.1.0. Also in the changelog

Reply:
I've changed the version to 0.1.0 in [fc338df](https://github.com/icapps/flutter_key_value_file_storage/pull/14/commits/fc338df85ffa6396a7aa96e6ed742b82da427b65)